defaults/main.yml

```diff
@@ -1,4 +1,4 @@
-crayfish_version_tag: 0.0.8
+crayfish_version_tag: master
```

Comment:
You can make this `0.0.9` as @dannylamb cut a release with the postgresql migrations in it.
https://github.com/Islandora-CLAW/Crayfish/tree/0.0.9